docs/errorhandling.rst

```diff
@@ -28,6 +28,59 @@ exception to the :attr:`~flask.Flask.logger`.
 But there is more you can do, and we will cover some better setups to deal
 with errors.
 
+.. _error-handlers:
+
+Error handlers
+--------------
+
+.. versionadded:: 1.0
+
+Some errors are unavoidable because not only you make them.  Users might
```

Comment:
true, especially the point about wanting to install them anyway (especially since that’s why i put it as first section)